src/LightNap.Core/Extensions/ApplicationUserExtensions.cs

```diff
@@ -49,7 +49,7 @@ public static void UpdateLoggedInUser(this ApplicationUser user,
         /// <returns>An AdminUserDto object representing the ApplicationUser object.</returns>
         public static AdminUserDto ToAdminUserDto(this ApplicationUser user)
         {
-            long? lockoutEnd = (user.LockoutEnd.GetValueOrDefault(DateTime.MinValue) < DateTime.UtcNow) ? null : user.LockoutEnd?.ToUnixTimeMilliseconds();
+            long? lockoutEnd = (user.LockoutEnd.GetValueOrDefault(DateTimeOffset.MinValue) < DateTime.UtcNow) ? null : user.LockoutEnd?.ToUnixTimeMilliseconds();
```

Comment:
Nice catch. Would you mind updating the comparison just to stay consistent with DateTimeOffset and I'll pull?

```suggestion
            long? lockoutEnd = (user.LockoutEnd.GetValueOrDefault(DateTimeOffset.MinValue) < DateTimeOffset.UtcNow) ? null : user.LockoutEnd?.ToUnixTimeMilliseconds();
```